examples/cifar10/predict.py

```diff
@@ -52,7 +52,7 @@ def predict(net, images, dev, topk=5):
 def load_dataset(filepath):
     print('Loading data file %s' % filepath)
     with open(filepath, 'rb') as fd:
-        cifar10 = pickle.load(fd)
+        cifar10 = pickle.load(fd, encoding='latin1')
```

Comment:
do we need to use latin1 if the file was dumped using pickle without specify the encoding field?